SQF/dayz_code/stringtable.xml

```diff
@@ -12895,7 +12985,7 @@
 		</Key>
 		<Key ID="STR_EPOCH_PLAYER_303">
 			<English>Murder</English>
-			<German>Mord</German>
+			<German>Ermordungen</German>
```

Comment:
I would choose "Morde".

Reply:
What about 'Tötungen'?

Reply:
That's your choice at the end but in my opinion "Morde" still sounds the best.